### PR TITLE
Feat: add Footer-component

### PR DESCRIPTION
**Pull request by Charbel-t**

### CODE REVIEW

File: src/Components/Footer/Footer.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  background: #0091ea;
+}
+
+.textColor {
+  color: #fff0f0;
+}
+
+.alignmentRow1 {
+  text-align: left;
+  padding-left: 4vw;
```
Comment (osamaakb) [inline src/Components/Footer/Footer.css]: don't use the unit `vw` with padding, sometimes the view is big or small and mostly you want a specific padding not a changed one depending on the screen

File: src/Components/Footer/Footer.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+              </Title>
+            </Col>
+            <Col span={8} />
+          </Row>
+          <br />
+          <Row>
+            <Col span={2} />
+            <Col span={20} className="alignmentRow2">
+              <Text className="textColor">
+                {' '}
```
Comment (osamaakb) [inline src/Components/Footer/Footer.js]: remove this `{'  '}`

File: src/Components/Footer/Footer.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+                Copyright © WeDot Wireframe Kit. All rights reserved
```
Comment (osamaakb) [inline src/Components/Footer/Footer.js]: change this to something real

**Review by osamaakb — CHANGES_REQUESTED** [review]

File: src/Components/Footer/Footer.css (hunk 1 above)
Comment (Charbel-t) [reply src/Components/Footer/Footer.css]: Okay, changes are done, check it now please.

**Review by osamaakb — APPROVED** [review]

**Pull request merged**